### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by fpob**
* added help to commands
* update example and docker usage
* removed duplicate sections, ie. variables, help text

Related to #9

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -68,17 +59,16 @@
...
+    * `TOWER_HOST`, `TOWER_USERNAME` and `TOWER_PASSWORD`
+    * `TOWER_CERTIFICATE` if you are using custom CA or self-signed
       certificates, value should be path to certificate or you can use
       [variable of type
       File](https://docs.gitlab.com/ee/ci/variables/#custom-environment-variables-of-type-file)
 1. Add assets (YAML files) to the repository (folder, eg. `data/assets/`)
-    * You can download the already existing ones with `podman run --rm -v $PWD:/workdir:Z europ/atacac backup <LABEL_ID> /workdir/data/assets`
-1. Configure the validation scheme stored in `data/schemas/`
-    * You can check asset validating schema locally via `podman run --rm -v $PWD:/workdir:Z -e ASSETS_GLOB="/workdir/data/assets/*.yml" -e ASSETS_SCHEMA=/workdir/<schema-path> europ/atacac validate`
+    * You can download the already existing ones with `atacac backup <LABEL_ID> 'data/assets/*.yml'`
```
Comment (europ) [inline docs/README.md:68]: This command has its prerequisites. You should add a reference to the section that describes how to run this *atacac*-command via container or installed *atacac* pip package.

File: docs/README.md [hunk 2]
```diff
@@ -68,17 +59,16 @@
...
+1. Configure the validation schema stored in `data/schemas/`
+    * You can check asset validating schema locally via `atacac validate 'data/assets/*_<type>.yml' 'data/schemas/<type>.yml'`
```
Comment (europ) [inline docs/README.md:70]: The same here.

**Review by europ — COMMENTED** [review]
Very nice job :+1: just one little note

File: docs/README.md (hunk 1 above)
Comment (fpob) [reply docs/README.md:68]: `pip3 install --user atacac` in "Prerequisites" section is not enough?

Comment (europ) [reply docs/README.md:68]: It should be enough. Hopefully, it is not necessary to mention it here.

**Review by europ — APPROVED** [review]

**Pull request merged**